rpi/config.nix

```diff
@@ -16,8 +16,7 @@ let
           (lib.filterAttrs (k: v: v.enable) x);
       render-dt-overlay = { overlay, args }:
         "dtoverlay=" + overlay + "\n"
-        + lib.strings.concatMapStringsSep "\n" render-dt-param args + "\n"
-        + "dtoverlay=";
```

Comment:
Why was this removed? It was there for a reason: it ends the `dtoverlay` parameter scope.

Reply:
I was getting an extra empty dtoverlay= in the generated config.

Reply:
I see how that would look like an error, but that line is meaningful; It delimits the scope of the parameters associated with the above overlay. For example:

```
dtoverlay=<SOME_OVERLAY>
param0=val0
param1=param1
...
dtoverlay=
```
This ensures that all params listed above are interpreted as parameters to `<SOME_OVERLAY>`, and avoids ambiguity if a parameter name occurs both in an overlay and a base config.

See the [docs](https://www.raspberrypi.com/documentation/computers/configuration.html#part3.1).

specifically, this part:
<img width="930" alt="Screenshot 2024-06-22 at 6 35 51 PM" src="https://github.com/tstat/raspberry-pi-nix/assets/1627482/f1c6fb0e-a291-4573-a196-b8ffce286d6f">

Reply:
undid that, thanks for the explanation!